.browserslistrc

```diff
@@ -1,9 +1,21 @@
-# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
```

Comment:
Maybe instead of a .browserslistrc, we can just use the [same browserlist](https://github.com/openmrs/browserslist-config-openmrs) we use everywhere else? 

In other places do this by adding it as a devDependency and then having this in the package.json:

```
"browserslist": [
    "extends browserslist-config-openmrs"
  ],
```

I'd assume Angular works with that too? (Also no need to support IE 11)